.github/workflows/ci-pr.yml

```diff
@@ -9,6 +9,7 @@ jobs:
       image: racket/racket-ci:latest
       options: --init
 
+    # 20.04?
```

Comment:
@pmatos

Can we use 20.04 consistently for everything? Or perhaps 18.04 consistently for everything. (I vote for 20.04 though. 18.04 is pretty old now)

Reply:
yeah, 20.04 sounds better at this point.